Source/Diagnostics/ComputeDiagFunctors/DivBFunctor.cpp

```diff
@@ -3,8 +3,8 @@
 
 using namespace amrex;
 
-DivBFunctor::DivBFunctor(const std::array<const amrex::MultiFab* const, 3> arr_mf_src, const int lev, const int ncomp)
-    : ComputeDiagFunctor(ncomp), m_arr_mf_src(arr_mf_src), m_lev(lev)
+DivBFunctor::DivBFunctor(const std::array<const amrex::MultiFab* const, 3> arr_mf_src, const int lev, const amrex::IntVect diag_crse_ratio, const int ncomp)
+    : ComputeDiagFunctor(ncomp, diag_crse_ratio), m_arr_mf_src(arr_mf_src), m_lev(lev)
 {}
```

Comment:
Thanks for all the suggestions, @MaxThevenet . Since diag_crse_ratio is change to m_crse_ratio, I could not directly commit the suggestions. But I have incorporated the suggested changes with `m_crse_ratio`.